pkg/login/social/azuread_oauth.go

```diff
@@ -20,19 +22,33 @@ type SocialAzureAD struct {
 }
 
 type azureClaims struct {
-	Email             string   `json:"email"`
-	PreferredUsername string   `json:"preferred_username"`
-	Roles             []string `json:"roles"`
-	Groups            []string `json:"groups"`
-	Name              string   `json:"name"`
-	ID                string   `json:"oid"`
+	Email             string                 `json:"email"`
+	PreferredUsername string                 `json:"preferred_username"`
+	Roles             []string               `json:"roles"`
+	Groups            []string               `json:"groups"`
+	Name              string                 `json:"name"`
+	ID                string                 `json:"oid"`
+	ClaimNames        claimNames             `json:"_claim_names,omitempty"`
+	ClaimSources      map[string]claimSource `json:"_claim_sources,omitempty"`
+}
+
+type claimNames struct {
+	Groups string `json:"groups"`
```

Comment:
Will we need this for Roles as well? 🤔

Reply:
There is nothing in the docs about roles overage. Seem like this only applies to groups